src/Kovaleva.HomeWork-4/Task.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+
+namespace Kovaleva.HomeWork_4
+{
+    public class Task
+    {
+        private readonly string Id;
```

Comment:
let's use a common naming convention for .net where private fields start from lower case character